### ISSUE TITLE
Error: Unable to locate file containing name

### ISSUE THREAD

**Issue by mscribe** [shi/crpm#8]
In CDK app.ts:

```typescript
new SomeStack(app, 'somename', {
  stackName: 'some-stack-name'
});
```

If you try `crpm i management-governance/cloudformation/stack -s somename`, it will output:
Unable to locate file containing somename

**Comment by mscribe** [comment]
Fixed in https://github.com/shi/crpm/commit/652407db960b78c654bcdc0b15f04397bf8335e4

**Comment by mscribe** [comment]
Fixed in v2.2.0:

https://github.com/shi/crpm/releases/tag/v2.2.0
